lib/multiplexer/multiplexer.go

```diff
@@ -53,6 +53,19 @@ var (
 		"client source and destination addresses should be valid same TCP version non-nil IP addresses")
 )
 
+// PROXYProtocolMode controls behavior related to PROXY protocol.
+// Possible values:
+// - 'on': one PROXY header is accepted and required per incoming connection.
+// - 'off': no PROXY headers are allows, otherwise connection is rejected.
+// - (unspecified): one PROXY header is allowed, but not required. Connection is marked with source port set to 0.
```

Comment:
Refer why we added this mode and that it's incorrect and no-one should ever use it.
Either use on if your L4 LB sends proxy lines or off if you don't send proxy lines at all or no L4 load balancer exists